tasks/WaveSourceTask.hpp

```diff
@@ -31,10 +30,11 @@ namespace gazebo_usv{
     protected:
         std::string mModelName;
         gazebo::transport::NodePtr mNode;
-        gazebo::transport::PublisherPtr mWaveVelocityPublisher;
+        gazebo::transport::PublisherPtr mWaveAmplitudePublisher;
```

Comment:
According to the latest coding standard all variables must be snake_case and private/protected should start with m_